My feedback

jlunabacilio opened the pull request:
This is my practice

Commit:
Updated readme

Commit:
Added this

JAdrianPZ committed:
Mi Feeback

JAdrianPZ committed:
Movi mi feeback a la carpeta correcta

Commit:
Retroalimentacion Academia DevOps

Commit:
Add JorgeTonatiuhBarajasVazquez.md

douglax committed:
Merge pull request #1 from JAdrianPZ/feature/JAPZ

Approved - Feature/japz

douglax committed:
Merge pull request #2 from FernandoGutierrezG/feature/JFGG

Approved - Creacion de archivo xx.md

ArmandoS02 committed:
Subiendo mi Feedback

ulisespont committed:
Agregue feedback

Carosftk committed:
Agregue mi feedback

alanrmartinez committed:
Feedback de Alan Martinez en formato marckdown

AraceliVR93 committed:
Se agrego AraceliVazquezRamirez.md

AraceliVR93 committed:
Added AraceliVazquezRamirez.md

Commit:
Retroalimentacion Academia DevOps

douglax committed:
Merge pull request #6 from Caro-Love21/feature/CMC

Approved - Agregue mi feedback

douglax commented:
Merging directly into master is not a recommended practice, please request to merge into develop branch. A new PR maybe?

douglax committed:
Merge pull request #11 from Diego2426/feature/DAOG

Approved - Feature/daog

douglax committed:
Merge pull request #10 from AraceliVR93/feature/AVR

Approved - Feature/avr

douglax committed:
Merge pull request #9 from alanrmtz/feature/arms

Approved - Feedback de Alan Martinez en formato marckdown

douglax committed:
Merge pull request #7 from ArmandoS02/feature/ASC

Approved - Subiendo mi Feedback

douglax committed:
Merge pull request #5 from upont/feature/UAPD

Approved - Agregue feedback

douglax committed:
Merge pull request #4 from jorgetbv/develop

Approved - Add JorgeTonatiuhBarajasVazquez.md

Commit:
Creacion de archivo xx.md

MoisesTapia committed:
Mi Feedback

Commit:
Commit para archivo de retroalimentación

Commit:
Commit para archivo de retroalimentación 2

Commit:
Commit para archivo de retroalimentación 3

Commit:
Commit para archivo de retroalimentación 4

Commit:
Commit para archivo de retroalimentación 5

Commit:
Commit para archivo de retroalimentación 6

Commit:
Commit para archivo de retroalimentación 7

Commit:
Commit para archivo de retroalimentación 8

Commit:
Commit para archivo de retroalimentación 9

Commit:
Commit para archivo de retroalimentación 10

Commit:
Commit para archivo de retroalimentación 11

Commit:
Commit para archivo de retroalimentación 12

Commit:
Commit para archivo de retroalimentación 13

Commit:
Commit para archivo de retroalimentación f

aldosofttek committed:
AldoMorenoSanchez.md

onahump committed:
Creating my brach and my markdown file

douglax committed:
Merge pull request #16 from Aldomspow/feature/AMS

Approved - AldoMorenoSanchez.md

douglax committed:
Merge pull request #14 from onahump/feature/oswaldop

Approved - Creating my brach and my markdown file

douglax committed:
Merge pull request #13 from IanCRV95/feature/ICRV

Approved - Feature/icrv

Kavir91 committed:
Create kavirleonardocerongomez.md

douglax committed:
Merge pull request #17 from Kavir91/feature

Approved - Create kavirleonardocerongomez.md

douglax committed:
Merge pull request #12 from MoisesTapia/feature/MPOT

Approved - Mi Feedback

torresemma committed:
Se creo mi feedback

Debido a problemas con mi cuenta apenas pude alcanzar tu repositorio, me salían errores o bugs que tu repositorio estaba cerrado o inalcanzable.

douglax committed:
Merge pull request #18 from jesustorressol/patch-1

Approved / Se creo mi feedback
